packages/mdc-theme/_mixins.scss

```diff
@@ -18,22 +18,38 @@
 
 // Applies the correct theme color style to the specified property.
 // $property is typically color or background-color, but can be any CSS property that accepts color values.
-// $style should be one of the map keys in $mdc-theme-property-values (_variables.scss).
+// $style should be one of the map keys in $mdc-theme-property-values (_variables.scss), or a literal color value.
 @mixin mdc-theme-prop($property, $style, $important: false) {
-  @if not map-has-key($mdc-theme-property-values, $style) {
-    @error "Invalid style: '#{$style}'. Choose one of: #{map-keys($mdc-theme-property-values)}";
-  }
+  @if type-of($style) == "color" {
+    $value: $style;
```

Comment:
I guess `$value` is here for consistency with the `@else` branch below, but it seems kind of unnecessary here? Could easily just reference `$style` directly.